app/views/govuk_publishing_components/components/_metadata.html.erb

```diff
@@ -9,14 +9,21 @@
 
   other ||= nil
   inverse ||= false
+  inverse_compress ||= false
 
   direction_class = ""
   direction_class = " direction-#{direction}" if local_assigns.include?(:direction)
 
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
   classes = %w(gem-c-metadata)
   classes << "direction-#{direction}" if local_assigns.include?(:direction)
-  classes << "gem-c-metadata--inverse" if inverse
+
+  if inverse && inverse_compress
+    classes << "gem-c-metadata--inverse" if inverse and inverse_compress
```

Comment:
Does `inverse and inverse_compress` need to be checked here again if it already was on the line above?

Same for the elseif below it

Reply:
Whoops, yes, meant to take that out - that was a hangover from an earlier approach. It's just the `if` statement now.